### ISSUE TITLE
How to access to admin page?

### ISSUE THREAD

**Issue by imkimchi** [FuzzyHobbit/bustabit-webserver#9]
I can't see no ways to access to admin page.

**Comment by marooj** [comment]
http://27.102.70.169:3841/admin-giveaway

**Comment by imkimchi** [comment]
@marooj yeah I found it on routes file but I have no auth to access the page. How can I get admin auth?

**Comment by marooj** [comment]
admin page "The page you are looking for doesn't exist..."
i think this program didn't have admin's page and you must create this by himself

**Comment by marooj** [comment]
i am installed Apache2 on server then in view's directory in "bustabit-webserver/views/" you can work with jquery ajax and in php you can connection string to postgresql
